sale_report_hide_price/views/report_saleorder.xml

```diff
@@ -61,7 +50,7 @@
                                 <td class="text-right">-N.A.-</td>
                             </t>
                         </tr>
-                        <t t-foreach="doc._get_tax_amount_by_group()" t-as="amount_by_group">
+                        <t t-foreach="doc.amount_by_group" t-as="amount_by_group">
```

Comment:
explain each change done plz

Reply:
@JayVora-SerpentCS  This method _get_tax_amount_by_group is no more in v12. it has changed with amount_by_group.